cor/shared/src/main/scala/paymentsEmailVerification/models/api/StartEmailVerificationJourneyResponse.scala

```diff
@@ -18,25 +18,26 @@ package paymentsEmailVerification.models.api
 
 import paymentsEmailVerification.models.{DerivedJson, EmailVerificationStateError}
 import play.api.libs.json.OFormat
-import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder, JsonDecoder, JsonEncoder}
+import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder, JsonCodecConfiguration, JsonDecoder, JsonEncoder}
```

Comment:
Is JsonCodecConfiguration used somewhere?

Reply:
no, good spot - the compiler should have warned that it was unused. I've just pushed up a new commit with new compiler flags to make sure missing imports are flagged to us